src/System.Windows.Forms/src/System/Windows/Forms/Control.cs

```diff
@@ -10670,6 +10874,17 @@ protected virtual void SetVisibleCore(bool value)
                         | (value ? SET_WINDOW_POS_FLAGS.SWP_SHOWWINDOW : SET_WINDOW_POS_FLAGS.SWP_HIDEWINDOW));
             }
         }
+
+        static unsafe void PrepareDarkMode(HWND hwnd, bool darkModeEnabled)
+        {
+            BOOL value = darkModeEnabled;
+
+            PInvoke.DwmSetWindowAttribute(
+                hwnd,
+                DWMWINDOWATTRIBUTE.DWMWA_USE_IMMERSIVE_DARK_MODE,
+                &value,
+                (uint)sizeof(BOOL));
```

Comment:
Did we want to assert success here?
```suggestion
            PInvoke.DwmSetWindowAttribute(
                hwnd,
                DWMWINDOWATTRIBUTE.DWMWA_USE_IMMERSIVE_DARK_MODE,
                &value,
                (uint)sizeof(BOOL)).AssertSuccess();
```